topics/statistics/tutorials/classification_regression/tutorial.md

```diff
@@ -0,0 +1,215 @@
+
+>    ![data](images/regression_1.png "Regression fit through the targets.")
+
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+# Classification
+
+[Classification](https://en.wikipedia.org/wiki/Statistical_classification) task assigns a category/class to a sample by learning a decision boundary using a dataset. This dataset is called a training dataset and contains a class/category for each sample. The algorithm which performs this task is called a classifier. The training dataset contains "features" as columns and a mapping between these features and the target is learned for each sample. The performance of mapping is evaluated using test dataset. The test dataset contains only the feature columns and not the target column. The target column is predicted using the mapping learned on the training dataset. In this tutorial, we will use a classifier to train a model using a training dataset, predict the targets for test dataset and visualize the results using plots.
+
+## Data upload
+
+The datasets required for this tutorial contain 9 features of breast cancer which include the thickness of clump, cell-size, cell-shape and so on ([more information](https://github.com/EpistasisLab/penn-ml-benchmarks/tree/master/datasets/classification/breast-w)). In addition to these features, the training dataset contains one more column as `target`. It has a binary value (0 or 1) for each row. `0` indicates no breast cancer and `1` indicates breast cancer. The test dataset does not contain the `target` column. The third dataset contains the all the samples from test dataset but also the `target` column which would be needed to create plot showing comparison between actual and predicted targets.
+
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial
+> 2. Import the following datasets and choose the type of data as `tabular`.
+> 
+>    ```
+>    https://zenodo.org/record/1401230/files/breast-w_train.tsv
+>    https://zenodo.org/record/1401230/files/breast-w_test.tsv
+>    https://zenodo.org/record/1401230/files/breast-w_targets.tsv
+>    ```
+> 
+>    {% include snippets/import_via_link.md %}
+>
+> 3. Rename datasets to `breast-w_train`, `breast-w_test` and `breast-w_targets`.
+>
+>    {% include snippets/rename_dataset.md %}
+>
+{: .hands_on}
+
+## Train a classifier
+In this step, `SVM (Support vector machine)` classifier is trained using `breast-w_train` dataset. The last column of this dataset assigns a category for each row. The classifier learns a mapping between each row and its category. This mapping is called a trained model. It is used to predict the categories of unseen data (`breast-w_test`).
+
+> ### {% icon hands_on %} Hands-on: Train a classifier
+> 
+> **SVM Classifier (Support vector machine)** {% icon tool %} with the following parameters
```

Comment:
Can you format all tool parameter descriptions like this:

> 1. **SVM Classifier (Support vector machine)** {% icon tool %} with the following parameters
>     - {% icon param-select %} *"Select a Classification Task"*: `Train a model`
>     - {% icon param-select %} *"Classifier type"*: `Linear Support Vector Classification`
>     - {% icon param-select %} *"Select input type"*: `tabular data`
>     - {% icon param-file %} *"Training samples dataset"*: `breast-w_train`
>     - {% icon param-check %} *"Does the dataset contain header"*: `Yes`
>     - {% icon param-select %} *"Choose how to select data by column"*: `All columns but by column header name(s)`
>     - {% icon param-text %} *"Type header name(s)"*: `target`
>     - {% icon param-file %} *"Dataset containing class labels"*: `breast-w_train`
>     - {% icon param-check %} *"Does the dataset contain header"*: `Yes`
>     - {% icon param-select %} *"Choose how to select data by column"*: `Select columns by column header name(s)`
>     - {% icon param-text %} *"Select target column(s)"*: `target`

Thanks 😄

Reply:
I can not find the tools on  usegalaxy.eu. Is it normal?

Reply:
One remaining plotting tool has been added. Now, all the referenced tools are present on usegalaxy.eu. Additionally, to find the tools I have added a few comment sections in the tutorial

Reply:
Fixed the formatting :)

Reply:
Also added the workflows and `data-library.yaml`file.